### PR TITLE
Deduplicate replacements

### PR DESCRIPTION
**Pull request by i-am-the-slime**
This helps with #67

### CODE REVIEW

File: src/LanguageServer/IdePurescript/CodeActions.purs [hunk 1]
```diff
@@ -54,7 +54,7 @@ getActions documents settings (ServerState { diagnostics, conn: Just conn }) { t
 
     fixAllCommand text rebuildErrors = if length replacements > 0 then [ replaceAllSuggestions text docUri replacements ] else [ ]
       where
-      replacements = mapMaybe getReplacementRange rebuildErrors
+      replacements = nubByEq eq $ mapMaybe getReplacementRange rebuildErrors
```
Comment (i-am-the-slime) [inline src/LanguageServer/IdePurescript/CodeActions.purs:57]: This is the only non-whitespace change.

Comment (i-am-the-slime) [reply src/LanguageServer/IdePurescript/CodeActions.purs:57]: Somehow suggestions were appearing multiple times which lead to none being executed (I don't know why). Deduplicating this helped. We could also use `nub` but there was no `Ord` instance for `Range` and I tried to make the smallest change possible.

Comment (nwolverson) [reply src/LanguageServer/IdePurescript/CodeActions.purs:57]: Well hard to see how this could go wrong, at worst no change...

Comment (i-am-the-slime) [reply src/LanguageServer/IdePurescript/CodeActions.purs:57]: That's what I thought, too. I was thinking we could merge this in and then see if we can still reproduce the problem.

**Pull request merged**